editoast/openapi.yaml

```diff
@@ -8983,6 +8984,72 @@ components:
           format: int64
         line_name:
           type: string
+    SearchResultItemTrainSchedule:
+      type: object
+      description: A search result item for a query with `object = "trainschedule"`
+      required:
+      - id
+      - train_name
+      - labels
+      - rolling_stock_name
+      - timetable_id
+      - start_time
+      - schedule
+      - margins
+      - initial_speed
+      - comfort
+      - path
+      - constraint_distribution
+      - power_restrictions
+      - options
+      properties:
+        comfort:
+          type: integer
+          format: int64
```

Comment:
As discussed, we might want to expose that as an enum, but the implications is not in the scope of this PR. Let's keep it as an integer for the purpose of this PR. @leovalais let's discuss that.

Reply:
Agreed on exposing this as an enum. This would require a conversion step after searching a train schedule. I can see multiple ways to achieve that:

1. Add a post-processing step when searching for a ts
2. Change the macro to allow some form of mapping, like it's done in ModelV2
3. Figure out a way to use what's already implemented in ModelV2 and make the search response optionally interface itself with `(trait Model)::Row`.

In order to decide which is best (if any), how severe is exposing this `int` is? Will the frontend even use it? If not, maybe removing it from the response solves our problem. Otherwise do we see another use case?